examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>TYPO: the first category -&gt; the first category.</p><p>At point 3, I think in our scenario we do the reconstruct only on a "worker" (but all workers can do that reconstruct)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Alright typos fixed, I'll remember to paste everything into GDocs from now on :D </p><p>I've also added the two remarks about that only one worker does the constructing in our case and about the additional communication step. </p>